index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<header class="Header">
+     <a class="mylogo">MY logo</a>
+             <nav>
+            <a href="#"> </a>
+            <img href src="./image/logobuger.JPG">
+     </nav>
+</header>
```

Comment:
- [ ] [optional] I would suggest you please adjust the spacing by decreasing the margin so It looks nicer. See the image below for reference
### Design
![your](https://user-images.githubusercontent.com/81727988/205421959-459a5de5-b1e0-41d8-b54e-9a01a35e59e5.png)